test/unit/config/config_manage/1607_root_filters/config/galaxy.ini

```diff
@@ -1204,12 +1204,6 @@ use_interactive = True
 
 #amqp_internal_connection = sqlalchemy+sqlite:///./database/control.sqlite?isolation_level=IMMEDIATE
 
-# Galaxy real time communication server settings
-#enable_communication_server = False
-#communication_server_host = http://localhost
-#communication_server_port = 7070
-# persistent_communication_rooms is a comma-separated list of rooms that should be always available.
-#persistent_communication_rooms =
```

Comment:
These files aren't meant to be changed, they're the configuration at older versions of Galaxy used to unit tests things we've changed over time (e.g. for linting). I doesn't hurt anything because they're comments but I think the right thing to do is keep the files the same and mark these options as deprecated and dropped in configuration linting. Here is a commit that does that if I can talk you into rebasing that into this: https://github.com/galaxyproject/galaxy/commit/295628bad657f46fe231355e11fc8708ba38e6ec

Reply:
Yeah, I removed it all since it was just commented noise, but I don't have an opinion on this.  I'll incorporate 295628b if you want.